cdhweb/people/sitemaps.py

```diff
@@ -1,11 +1,26 @@
-from cdhweb.people.models import Profile
-from cdhweb.resources.sitemaps import PublishedItemSitemap
+from django.contrib.sitemaps import Sitemap
+from django.urls import reverse
 
+from cdhweb.people import views
 
-class ProfileSitemap(PublishedItemSitemap):
-    model = Profile
+
+class PeopleListSitemap(Sitemap):
+    '''Sitemap for people views that are not Wagtail pages but also
+    not tied to models (currently all people list views).'''
 
     def items(self):
-        # only published profiles with is_staff actually have detail pages
-        # and should appear in the sitemap
-        return super().items().filter(is_staff=True)
+        # return list of tuple with url name, view object
+        return [
+            ('staff', views.StaffListView),
+            ('students', views.StudentListView),
+            ('affiliates', views.AffiliateListView),
+            ('exec-committee', views.ExecListView)
```

Comment:
is it possible to import these from the urlconf so they're not hardcoded? or does that not make any sense

Reply:
I'm not sure how to import just the ones we care about. Did you have an idea about that? 

Agree that there is some duplication here, but if the views change I think we'd likely need to revisit the sitemaps as well.

After writing the blog list sitemap, I wonder if these could be implemented based more directly on the db models, but I don't want to revisit right now since it's at least working this way!